.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI checks
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Should we also run this when `push`? Since we don't require PR to be on the same base of `main` iirc.

Reply:
That's true!!!!

Reply:
Fixed in 9ddd4043